hr_employee_firstname/models/hr_employee.py

```diff
@@ -9,15 +9,34 @@
 
 _logger = logging.getLogger(__name__)
 
-UPDATE_PARTNER_FIELDS = ["firstname", "lastname", "user_id", "address_home_id"]
-
 
 class HrEmployee(models.Model):
     _inherit = "hr.employee"
 
+    @api.model
+    def _names_order_default(self):
+        return "first_last"
+
+    @api.model
+    def _get_names_order(self):
+        """Get names order configuration from system parameters.
+        You can override this method to read configuration from language,
+        country, company or other"""
+        return (
+            self.env["ir.config_parameter"]
+            .sudo()
+            .get_param("employee_names_order", self._names_order_default())
```

Comment:
Could you improve indentation please, remember min length now it's 119

Reply:
When I try to make this change, Travis CI shows the following error proposing as it was previously:

![Selección_925](https://user-images.githubusercontent.com/16093067/97561202-29258880-19b6-11eb-9d7b-388398c42e2b.png)


So I reversed the change

Reply:
@hugho-ad

The OCA guidelines need this structure and are based on the partner_contact module, I prefer this structure

![Screen Shot 2020-10-29 at 9 04 27](https://user-images.githubusercontent.com/7606656/97599574-c2ac6480-19cd-11eb-8cf9-701730a1cebc.png)